docs/source/notes/autograd.rst

```diff
@@ -87,6 +87,22 @@ subject to change and that users should not rely on.
 You can control how PyTorch does packing / unpacking with :ref:`saved-tensors-hooks-doc`.
 
 
+.. _non-differentiable-func-grad:
+
+Gradients for non-differentiable functions
+------------------------------------------
+
+The gradient computation using Automatic Differentiation is only valid when each elementary function being used is continuously differentiable.
```

Comment:
Why continuously differentiable?. Just differentiable is fine.

Reply:
Yep, my bad!